### ISSUE TITLE
When there is lots of content, pop-ups should not expand until they stretch off the screen, which then scrolls? There be scrollbars within the window.

### ISSUE THREAD

**Issue by wu-lee** [TransitionbyDesign/homemaker#26]

**Comment by wu-lee** [comment]
In the current scheme, pop-ups expand to cover the map, and contain content which will have a scrollbar if it doesn't fit on the display.

**Comment by wu-lee** [comment]
Pins now open pop-up windows, so if that is acceptable (and checking with @Charlie-TbD this seems to be), then this is not relevant, and this issue can be closed.

**Comment by Charlie-TbD** [comment]
Yes this has been dealt with under the curren budget but can we add this to a 'future improvements' list? I still prefer the functionality of the Who Owns Oxford website when it gets narrow and would prefer that if we return to tune certain elements.

![image](https://user-images.githubusercontent.com/72264494/98122239-96528580-1eb0-11eb-879c-07cccc28cd3b.png)
